ang/civicase/activity/card/directives/activity-card.directive.js

```diff
@@ -94,11 +94,15 @@
      * @param {object} activity
      */
     $scope.viewInPopup = function ($event, activity) {
+      var activityFormUrl = activity.case_id
+        ? 'civicrm/case/activity'
+        : 'civicrm/activity';
+
       if ($event && $($event.target).is('a, a *, input, button, button *')) {
         return;
       }
 
-      CRM.loadForm(CRM.url('civicrm/case/activity', {
+      CRM.loadForm(CRM.url(activityFormUrl, {
         action: 'update',
         id: activity.id,
         caseid: activity.case_id,
```

Comment:
Is the case id necessary to be sent when we are using non-case modal?

Reply:
it's not necessary, but it doesn't break the other form either since this value is sent as null anyway.

Reply:
Lets not send this please, it looks confusing that that why are we sending case info when it i not even there.